dali/test/python/test_operator_rotate.py

```diff
@@ -131,10 +149,25 @@ def warp_fn(img, angle):
 
 
 class RotatePipeline(Pipeline):
-    def __init__(self, device, batch_size, output_type, input_type, fixed_size=None, num_threads=3, device_id=0, num_gpus=1):
-        super(RotatePipeline, self).__init__(batch_size, num_threads, device_id, seed=7865, exec_async=False, exec_pipelined=False)
+    def __init__(self,
+                 device,
+                 batch_size,
+                 output_type,
+                 input_type,
+                 fixed_size=None,
+                 num_threads=3,
+                 device_id=0,
+                 num_gpus=1):
+        super(RotatePipeline, self).__init__(batch_size,
+                                             num_threads,
+                                             device_id,
+                                             seed=7865,
+                                             exec_async=False,
+                                             exec_pipelined=False)
```

Comment:
I think that argument-per-line is more readable - and I'd recommend this style for use with autoformatters, because at least this way it's impossible to split the argument list in the middle of a logical group.